### PR TITLE
Current python.

### PR DESCRIPTION
**Pull request by sallner**
Update to current Python versions in order to make a release really up to date.

### CODE REVIEW

**Review by jugmac00 — APPROVED** [review]
lgtm!

**Review by icemac — APPROVED** [review]
LGTM but what about adding `pypy3` as well?

File: setup.py [hunk 1]
```diff
@@ -49,6 +49,8 @@ def read(*rnames):
           'Programming Language :: Python :: 3.4',
```
Comment (stevepiercy) [inline setup.py]: Consider dropping support for EOL Python 3.4.

Comment (jugmac00) [reply setup.py]: Good catch!

Comment (sallner) [reply setup.py]: It was not tested in `.travis.yml` and `tox.ini`, so was blindly following the content. Thanks for the Review.

Comment (mgedmin) [reply setup.py]: Shameless plug: https://pypi.org/project/check-python-versions/ can catch inconsistencies like that (and automate Python version updates in all the files).

**Review by mgedmin — APPROVED** [review]

**Comment by sallner** [comment]
I have added PyPy 3.

File: setup.py (hunk 1 above)
Comment (sallner) [reply setup.py]: That looks very interesting and might end up in my package maintainer venv.

**Pull request merged**